src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -27,6 +27,76 @@ public sealed partial class ApplicationIdentity : System.Runtime.Serialization.I
     }
 }
 
+namespace System.Data.Common
+{
+    public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        [Obsolete]
```

Comment:
These are `[Obsolete(msg, true)]` in netfx.  We probably want to keep the error form of Obsolete...